tests/fixtures/saas/test_data/interruptible/interruptible_config.yml

```diff
@@ -0,0 +1,46 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Interruptible Service
```

Comment:
ah this again. So do you just need the above override so you can test it here? If so, that's fine, but we should def have a comment on that above file explaining this more.